desktop/core/src/desktop/js/onePageViewModel.js

```diff
@@ -442,13 +442,28 @@ class OnePageViewModel {
       $('.dz-drag-hover').removeClass('dz-drag-hover');
     };
 
+    let dbname = '';
+    let interpreter = '';
+
     const openImporter = function (path) {
+      const table = path.substring(path.lastIndexOf(':') + 1, path.lastIndexOf(';'));
       self.loadApp('importer');
       self.getActiveAppViewModel(vm => {
         vm.createWizard.source.path(path);
+        vm.currentStep(2);
+        vm.createWizard.source.interpreter(interpreter);
+        vm.createWizard.destination.name(dbname + '.' + table.substring(0, table.length - 4));
```

Comment:
Here can't we simulate via the guessFormat(), guessFieldTypes() then indexFile()?

Reply:
When we provide the path, interpreter and name then automatically those functions will get executed.